app/src/main/java/com/zulip/android/networking/GravatarAsyncFetchTask.java

```diff
@@ -47,7 +48,7 @@ public static URL sizedURL(Context context, String url, float dpSize) {
         float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP,
                 dpSize, r.getDisplayMetrics());
         try {
-            return new URL(url + "&s=" + px);
+            return new URL(UrlHelper.getSenderAvatarUrl(url)  + "&s=" + px);
```

Comment:
just to clarify: using new `getSenderAvatarUrl` func is redundant as it does probably the same job as UrlHelper.addHost
[RecyclerMessageAdapter.setUpGravatar](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java#L788) is where we use addHost for the same purpose.